operators/pkg/dev/portforward/pod_forwarder.go

```diff
@@ -177,7 +177,7 @@ func (f *PodForwarder) DialContext(ctx context.Context) (net.Conn, error) {
 		return nil, f.viaErr
 	}
 
-	log.Info("Redirecting dial call", "addr", f.addr, "via", f.viaAddr)
```

Comment:
I was about to remove those, but there's one hidden in client-go that we can't remove anyway. Turned them into debug instead 🤷‍♂